addons/mail/models/mail_activity.py

```diff
@@ -625,6 +625,10 @@ def activity_format(self):
                 {'id': attachment.id, 'name': attachment.name}
                 for attachment in record.attachment_ids
             ]
+            activity["thread"] = {
+                "id": activity["res_id"],
+                "model": activity["res_model"],
+            },
```

Comment:
Can't we remove `res_id` and `res_model` from result then?

Reply:
the `res_id` and `res_model` in result are provided to detect the result belongs to which thread in `this.store.Thread.insert(result)`

Reply:
But if thread is provided, the individual values for res_id and res_model are now duplicates?